gen_cert.py

```diff
@@ -102,7 +102,9 @@ class CertificateGen(object):
     """Manages the pdf, signatures, and S3 bucket for course certificates."""
 
     def __init__(self, course_id, template_pdf=None, aws_id=None, aws_key=None,
-                 dir_prefix=None, long_org=None, long_course=None, issued_date=None):
+                 dir_prefix=None, long_org=None, long_course=None, issued_date=None,
+        title=None,
```

Comment:
@caseylitton How are you invoking pep8? I suspect you may be doing something non-standard.
I get the same results as travis; just the 3 errors, not 70.